packages/smart-wallet/src/smartWallet.js

```diff
@@ -444,7 +444,7 @@ const SmartWalletKit = defineVirtualFarClassKit(
             facets.helper.publishCurrentState();
           },
         });
-        executor.executeOffer(offerSpec);
+        await executor.executeOffer(offerSpec);
```

Comment:
seems like a substantive change.

But I suppose clients use chainStorage to sense progress / completion, so they won't notice.